src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePicker.cs

```diff
@@ -123,7 +160,7 @@ private void SetupFlyoutButton()
 			UpdateDisplayedDate();
 		}
 
-		partial void OnTimeChangedPartial(TimeSpan oldTime, TimeSpan newTime)
+		void OnTimeChangedPartial(TimeSpan oldTime, TimeSpan newTime)
```

Comment:
Please always specify the access modifier
```suggestion
		private void OnTimeChangedPartial(TimeSpan oldTime, TimeSpan newTime)
```